### PR TITLE
add back pybullet skills in pythonic format

### PR DESCRIPTION
**Pull request by tomsilver**
we no longer need all the task planning stuff in this new framing.

output of test:
https://github.com/user-attachments/assets/d297dc4b-b9bb-4d08-97e4-039f07bb122b

### CODE REVIEW

File: src/multitask_personalization/envs/pybullet/pybullet_skills.py [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+def generate_surface_placements(
+    surface_id: int, obj_id: int, sim: PyBulletEnv, rng: np.random.Generator
+) -> Iterator[Pose]:
+    """Sample placements uniformly on the top of the given surface."""
+    surface_extents = get_aabb_dimensions(surface_id, sim)
+    object_extents = get_aabb_dimensions(obj_id, sim)
+    placement_lb = (
+        -surface_extents[0] / 2 + object_extents[0] / 2,
+        -surface_extents[1] / 2 + object_extents[1] / 2,
+        surface_extents[2] / 2 + object_extents[2] / 2,
```
Comment (RealZiangLiu) [inline src/multitask_personalization/envs/pybullet/pybullet_skills.py:114]: Same as above, would this cause issues if the object is placed "inside" the surface object (like a shelf)?

File: src/multitask_personalization/envs/pybullet/pybullet_skills.py [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    GripperAction,
+    PyBulletAction,
+    PyBulletState,
+)
+from multitask_personalization.rom.models import (
+    ROMModel,
+)
+
+
+def get_object_id_from_name(object_name: str, sim: PyBulletEnv) -> int:
```
[the remark this answers is not in the document]
Comment (tomsilver) [reply src/multitask_personalization/envs/pybullet/pybullet_skills.py:37]: good call on making it a member function, I will do that. for `sim`, I want to leave it named that way to emphasize that the PyBulletEnv here is actually a simulator, not the real environment. whenever I use `env` I mean that to be the "real" environment

File: src/multitask_personalization/envs/pybullet/pybullet_skills.py [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+    object_id: int, sim: PyBulletEnv, distance_threshold: float = 1e-3
+) -> int:
+    """Get the PyBullet ID of the surface that the object is on."""
+    surfaces = get_surface_ids(sim)
+    assert object_id not in surfaces
+    object_pose = get_pose(object_id, sim.physics_client_id)
+    for surface_id in surfaces:
+        surface_pose = get_pose(surface_id, sim.physics_client_id)
+        # Check if object pose is above surface pose.
+        if object_pose.position[2] < surface_pose.position[2]:
```
[the remark this answers is not in the document]
Comment (tomsilver) [reply src/multitask_personalization/envs/pybullet/pybullet_skills.py:66]: yeah this is a very good point -- I don't see an easy way to generalize it right now, so I'll just add a note in the code

File: src/multitask_personalization/envs/pybullet/pybullet_skills.py (hunk 1 above)
Comment (tomsilver) [reply src/multitask_personalization/envs/pybullet/pybullet_skills.py:114]: good point -- added note

**Pull request merged**